### ISSUE TITLE
Show "in progress" status in PRs?

### ISSUE THREAD

**Issue by FezVrasta** [siddharthkp/bundlesize#9]
Right now I see the bundlesize report only when the PR is completed. I'd love to see it in "yellow" while the build is still proceeding.

Maybe add a `bundlesize start` command to run before any other build command to notify the PR about this check?

The `test` script would look like this:

```
"scripts": {
  // better command names are welcome 🙄
  "test": "bundlesize start && jest && bundlesize check"
}
```

**Comment by siddharthkp** [comment]
`bundlesize` does add a pending build but it's pretty fast so you almost never see it 😋 

I can totally add a explicit command to start the build. Not sure if it would be useful though

**Comment by FezVrasta** [comment]
I see what you mean, but the thing is that if I run `bundlesize` before I run my tests & build, it will never find the bundle file.

So my `test` script is:

```
"test": "lerna run --parallel test && npm run build && bundlesize",
```

Because I first run the tests, then I create a build and then `bundlesize` can read the size of it.

While I wait for `test` and `build` to complete, there's not a bundlesize "in progress" badge

**Comment by siddharthkp** [comment]
Yep, I totally get that.

Tell you what, I'll add it if > 5 people want it. That's fair? 😄

**Comment by siddharthkp** [comment]
@FezVrasta You got your 5 votes, so here it is 😄 

`bundlesize` ships 2 commands now: `bundlesize` and `bundlesize-init`

You can run `bundlesize-init` first, that will set a pending status on the build.
Then build your bundles, and call `bundlesize`.

This is published in `bundlesize@0.11.0`

**Comment by FezVrasta** [comment]
Cool thank you. I'll give it a try
